catalog/templates/catalog/product_form.html

```diff
@@ -3,43 +3,15 @@
 {% block title %}Добавление товара{% endblock %}
 {% block content %}
 
-<div class="container m-3">
-    {% if existing == True %}<h1>Товар уже существует в базе</h1>
-    {% elif existing == False %}<h1>Товар успешно добавлен в базу</h1>
-    {% else %} <h1>Внесите реквизиты нового товара в форму</h1>
-    {% endif %}
-</div>
-
 <div class="container">
     <form method="post" class="col-4" enctype="multipart/form-data">
         {% csrf_token %}
         {{ form.as_p }}
-
-<!--        <div class="mb-3">-->
-<!--            <label for="name" class="form-label">Наименование</label>-->
-<!--            <input type="text" class="form-control" id="name" name="name">-->
-<!--        </div>-->
-<!--        <div class="mb-3">-->
-<!--            <label for="textarea" class="form-label">Описание</label>-->
-<!--            <textarea class="form-control" name="textarea" id="textarea"></textarea>-->
-<!--        </div>-->
-<!--        <div class="mb-3">-->
-<!--            <label for="disabledSelect" class="form-label">Категория товара</label>-->
-<!--            <select id="disabledSelect" class="form-select" name="category">-->
-<!--                {% for category in categories %}-->
-<!--                <option>{{category.name}}</option>-->
-<!--                {% endfor %}-->
-<!--            </select>-->
-<!--        </div>-->
-<!--        <div class="mb-3">-->
-<!--            <label for="price" class="form-label">Стоимость</label>-->
-<!--            <input type="number" class="form-control" id="price" name="price">-->
-<!--        </div>-->
-<!--        <div class="mb-3">-->
-<!--            <label for="formFile" class="form-label">Фото товара</label>-->
-<!--            <input class="form-control" type="file" name='file' id="formFile">-->
-<!--        </div>-->
-        <button type="submit" class="btn btn-primary">Добавить</button>
+        <button type="submit" class="btn btn-primary">
+            {% if change == True %}Изменить
```

Comment:
nit: сравнивать буль с True/False не нужно
```suggestion
            {% if change  %}Изменить{% else %}Добавить{% endif %}
```